MergerLogic/Utils/ImageUtils.cs

```diff
@@ -6,15 +6,28 @@ public class ImageUtils
     {
         public static bool IsTransparent(MagickImage image)
         {
-            if(!image.HasAlpha) {
+            if (!image.HasAlpha)
+            {
                 return false;
             }
 
             using var pixels = image.GetPixels();
             // Check pixels to see if all are transparent
```

Comment:
Could you please fix the comment? It should be `if at least one pixel is transparent`.

Reply:
fixed